Yafc/Workspace/ProductionTable/ProductionLinkSummaryScreen.cs

```diff
@@ -39,8 +39,92 @@ private void BuildScrollArea(ImGui gui) {
             gui.BuildText((amount > 0 ? "Overproduction: " : "Overconsumption: ") + DataUtils.FormatAmount(MathF.Abs(amount), link.flowUnitOfMeasure),
                 new TextBlockDisplayStyle(Font.subheader, Color: SchemeColor.Error));
         }
+        ShowRelatedLinks(gui);
     }
 
+    private void ShowRelatedLinks(ImGui gui) {
+        ModelObject o = link;
+        List<ModelObject> parents = [];
+        while (o.ownerObject is not ProjectPage) {
+            var t = o.ownerObject;
+            if (t is null) {
+                break;
+            }
+            o = t;
+            if (t == o.ownerObject) continue;
+            parents.Add(t);
+        }
+        if (o is ProductionTable page) {
+            Dictionary<ProductionLink, List<(RecipeRow row, float flow)>> childLinks = [], parentLinks = [], otherLinks = [];
+            List<(RecipeRow row, float flow)> unlinked = [], table;
+            foreach (var (r, relationLinks) in
```

Comment:
as mentioned elsewhere: `r` is for recipe? If so, I would rename. It took me a while to figure out, making the code less readable (for me at least).

Reply:
actually it's `row`, but `recipe` also works. The type is `recipeRow` xD